File: masar_cable/custom/price_list/price_list.py
import frappe
import erpnext
import math
from frappe.utils import flt, get_datetime, getdate, date_diff, cint, nowdate, get_link_to_form, time_diff_in_hours
from frappe import throw, msgprint, _

@frappe.whitelist()
def apply_vfcu(doctype,name,lme1,lme2):
    pl_doc = frappe.get_doc(doctype,name)

    for d in frappe.db.get_list('Item Price',filters={'price_list': name},fields=['name','item_code']):
        item_doc = frappe.get_doc("Item", d.item_code)
        item_price_doc = frappe.get_doc("Item Price", d.name)
        item_price_doc.price_list_rate = item_price_doc.price_list_rate + item_doc.vfcu * (flt(lme1) - flt(lme2))
        item_price_doc.save()

File: masar_cable/custom/price_list/price_list.js

frappe.ui.form.on ("Price List", "apply_vfcu", function(frm) {
  if(!frm.doc.current_lme) return;
  frm.set_value("lme2",frm.doc.lme1);
  frm.set_value("lme1",frm.doc.current_lme);
   frappe.call({
        method:"masar_cable.custom.price_list.price_list.apply_vfcu",
        args:{"doctype": "Price List","name":frm.doc.name,"lme1":frm.doc.lme1,"lme2":frm.doc.lme2},
        callback: function(r) {
          cur_frm.save();
          show_alert("Prices are changed",5);
        }
    });

  });
